bandcamp_dl/bandcampdownloader.py

```diff
@@ -58,7 +59,7 @@ def start(self, album: dict):
         if self.debugging:
             logging.basicConfig(level=logging.DEBUG)
 
-        if album['full'] is not True:
+        if not album['full'] and not self.confirmation_skip:
```

Comment:
The reason for "renaming" no_confirm to confirmation_skip was to have this line "read nicely". Also I changed `is not True` to just `not`, I think that's more pythonic and fits better with the new and longer statement.